src/dynamics.js

```diff
@@ -111,7 +111,9 @@ function handleClick() {
     return;
   }
   document.getElementById('navbarPrimary').addEventListener('click', function(event) {
-    button.click();
+    if(getComputedStyle(button).display !== 'none'){
```

Comment:
Not sure about the indentation and whitespace standards in this project, if any.

Shouldn't we add a space after `if`, and before `{`?

Reply:
we could add prettier or something, but I think it could be another issue

Reply:
Good enough for me. I just noticed the surrounding code does have spaces in these places.

Reply:
I use the builtin formatter of vscode and for js/ts does just the bare minimum, so it will not add/remove the spaced there.

Reply:
Ah, fair enough.

Reply:
opened https://github.com/falconry/falconry.github.io/issues/63 just as a reminder if we want to go that way